docs/sources/flow/reference/cli/run.md

```diff
@@ -63,26 +63,35 @@ reloading.
 
 [component controller]: {{< relref "../../concepts/component_controller.md" >}}
 
-## Clustered mode (experimental)
+## Clustering (beta)
```

Comment:
Heads up: there should be a changelog entry indicating that clustering in Flow mode has graduated to beta.